src/main/webapp/resources/js/pages/projects/share/TargetMetadataRestriction.jsx

```diff
@@ -0,0 +1,115 @@
+import { Form, Popover, Select, Space, Tag, Tooltip, Typography } from "antd";
+import React from "react";
+import { useDispatch } from "react-redux";
+import {
+  IconInfoCircle,
+  IconWarningOutlined,
+} from "../../../components/icons/Icons";
+import { blue6, red6 } from "../../../styles/colors";
+import {
+  getColourForRestriction
+} from "../../../utilities/restriction-utilities";
+import { updateMetadataRestriction } from "./shareSlice";
+
+/**
+ * React component to allow the user to select the level of restiction for a
+ * metadata field in the destination project.
+ *
+ * @param {object} field - field to set value on
+ *  project, if it is not in the target project it get the current restriction.
+ * @param {array} restrictions - list of available restrictions
+ * @param {function} onChange - change handler
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function TargetMetadataRestriction({ field = {}, restrictions = [] }) {
+  const dispatch = useDispatch();
+
+  const [feedback, setFeedback] = React.useState({
+    hasFeedback: false,
+    validateStatus: "",
+  });
+
+  const [tooltipVisible, setTooltipVisible] = React.useState(false);
+
+  React.useEffect(() => {
+    if (field.difference > 0) {
```

Comment:
Add a comment that `difference`  is the difference in the restriction levels.

Reply:
Updated in 7ca38bd52fb560a4feaf1d0ed4af97d2e4b33d73